### ISSUE TITLE
"The Expo SDK requires Expo to run" on Android right after expo init

### ISSUE THREAD

**Issue by dkaufhold** [expo/expo#2625]
Not entirely sure if it's an issue for expo or expo-cli. 

### Environment
```
  Expo CLI 2.2.6 environment info:
    System:
      OS: macOS 10.14
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 10.11.0 - /usr/local/bin/node
      Yarn: 1.9.4 - /usr/local/bin/yarn
      npm: 6.4.1 - /usr/local/bin/npm
      Watchman: 4.7.0 - /usr/local/bin/watchman
    IDEs:
      Android Studio: 3.2 AI-181.5540.7.32.5014246
      Xcode: 10.1/10B61 - /usr/bin/xcodebuild
    npmPackages:
      expo: ^31.0.2 => 31.0.2 
      react: 16.5.0 => 16.5.0 
      react-native: https://github.com/expo/react-native/archive/sdk-31.0.0.tar.gz => 0.57.1 
      react-navigation: ^2.18.2 => 2.18.2 
    npmGlobalPackages:
      expo-cli: 2.2.6

  Diagnostics report:
    https://exp-xde-diagnostics.s3.amazonaws.com/mobility-budget-a4f4fabf-ba7d-4c17-a462-95ad68cda44e.tar.gz
```
### Steps to Reproduce

1. with the latest Expo CLI run `expo init expotest`
2. open Android emulator - i.e. Pixel 2 API 28
3. run expo start
4. press `a` to open app in emulator

### Expected Behavior

App starts and shows home screen

### Actual Behavior

It shows this screen:

![screenshot_1541500821](https://user-images.githubusercontent.com/963102/48059204-cdeea000-e1b8-11e8-971a-509b8c8b1716.png)

### Reproducible Demo

Although it's literally just the state right after running `expo init`, here's the demo repo:

https://github.com/dkaufhold/expotest

### What I've tried

The solutions from this issue https://github.com/expo/expo/issues/2058

* made sure `app.json` contains right version
* the `expo-cli` was just updated before creating the project

**Comment by sjchmiela** [comment]
Hey! I couldn't reproduce your issue (I've cloned your repo and installed dependencies with `yarn`). Maybe you could try upgrading `expo` to `31.0.4` or ensuring that the latest version of Expo Client is installed on your emulator (`expo install:android`)?

**Comment by dkaufhold** [comment]
Upgrading `expo` to `31.0.4` made the error disappear. Thanks a lot.

**Comment by vayurobins** [comment]
Hi, I have the same issue and am running expo 31.0.4, but the problem is still there. I have also tried to install the latest expo client, latest expo-cli and app.js is using "sdkVersion": "31.0.0".

**Comment by coulix** [comment]
I am still getting this error with SDK32 with a freshly ejected Expokit app. Android phone has latest Expo client installed.

The app boots then displays this screen once loading screen is ready.

**Comment by HossamSamir** [comment]
same issue using 
`
"expo": "^32.0.0", 
"react": "16.5.0",
    "react-native": "https://github.com/expo/react-native/archive/sdk-32.0.0.tar.gz"
`
and `"sdkVersion": "32.0.0",`

**Comment by SuperKMan** [comment]
I got the same issue and I solved by:
run 'expo build:android' after that 'npm start'  again.
Hope this help.

**Comment by clayrab** [comment]
This is still a problem

**Comment by brentvatne** [comment]
@clayrab - can you create a new issue and provide a reproducible example project? thanks!

**Comment by mineshpatel1** [comment]
I'm experiencing this too, on Expo SDK 32, Expo CLI 2.13.0, I think it's premature to close this issue.

I was using Expo just fine, until I built an Android app. Since then, I can't run it on the emulator from my laptop, although it still works from my real phone.

**Comment by kevincolten** [comment]
I finally got it to start working again by updating to Expo SDK 32.0.1, Expo CLI 2.14.0, and restarting my computer.

**Comment by mohit23x** [comment]
delete package-lock.json and node-modules folder, and make sure your package.json has some expo version specified and it is not blank. In my case the expo version is  `"expo": "^32.0.0",` and then run `npm install` in the terminal.

**Comment by msxrx** [comment]
Notice the Wi-Fi icon in the statusbar. We've realized that if the emulator has no internet connectivity while installing the Expo client the app fails to run. In our case the connectivity problems were caused by a misconfigured office DNS server. As soon as we switched the DNS on our development machines to 8.8.8.8 and restarted the Android SDK the problems were gone.

We're on v33
